Core/Sources/Implementations/gene.cpp

```diff
@@ -1,3 +1,9 @@
+#ifdef _WIN32
```

Comment:
- Move this to utilites.hpp so we have DRY code.  
- Use a name that is guaranteed to not cause collisions with usermade defitions, like `HIPPOCRATES_SSCANF`